pennylane/ops/qubit/matrix_ops.py

```diff
@@ -735,3 +722,49 @@ def label(
         cache: Optional[dict] = None,
     ):
         return super().label(decimals=decimals, base_label=base_label or "BlockEncode", cache=cache)
+
+
+def _process_blockencode(*params, **hyperparams):
```

Comment:
Can we be a bit more explicit with the call signature here?

Reply:
Just curious if I missed a hyperparameter?

But this was just a nice-to-have while we were already changing the code, but we can just come back to this as part of the technical roadmap later.

Reply:
Yeah let's backlog and revisit later. In this PR changes are only to move things around for readability.

Reply:
[sc-89854]